### ISSUE TITLE
[Form A11y] Form errors should be marked with `role="alert"` and `aria-live="assertive"`

### ISSUE THREAD

**Issue by zoglo** [contao/contao#9176]
### Description

- Error messages should be marked with role="alert" and aria-live="assertive"

**Comment by zoglo** [comment]
Created a sub issue of the original parent issue in #6976 to better track the multiple and different features that have been requested by @lizardat
